qa/common/gen_qa_sequence_models.py

```diff
@@ -1108,10 +1108,14 @@ def create_libtorch_modelfile(models_dir, model_version, max_batch, dtype,
                               shape):
 
     if not tu.validate_for_libtorch_model(dtype, dtype, dtype, shape, shape,
-                                          shape):
+                                          shape, max_batch):
         return
 
+    # Skip for String I/O
     torch_dtype = np_to_torch_dtype(dtype)
+    if torch_dtype is None:
+        return
+
```

Comment:
You can check type and skip model / config generation at where these two functions get called in all the gen files, i.e. ~line 1402, `if FLAGS.libtorch and xxx:`

Reply:
Done